app/(editor)/editor/page.tsx

```diff
@@ -185,9 +65,13 @@ export default async function Page({ searchParams }: PageProps) {
 
   // Get data out of the API.
   const { data } = await getStoryData(searchParams);
-  let storyList;
-  if (slug === 'stories' || slug.includes('stories/list/')) {
-    storyList = await getStoryList(searchParams);
+
+  // An extra container for passing along additional fetched data.
+  let extra = {};
```

Comment:
I renamed the prop just to make it super generic so that it can be a place for future components that need additional data to put their data.